components/Website_Layout/Navbar.js

```diff
@@ -26,6 +41,17 @@ export default function navbar() {
       <Link href="/MerchandisePage">
         <a style={tabStyle}>Buy our Merchandise!</a>
       </Link>
+      {loggedIn ? <p>{username}</p>:
+      <GoogleLogin
+        render={renderProps => (
+          <button onClick={renderProps.onClick} disabled={renderProps.disabled}>Login</button>
```

Comment:
I deployed to qa and it looks like the button on line 47 is always disabled. Double check` renderProps.disabled`

Reply:
yea u gotta have a seperate oauth key for deployment. current one only works for localhost

Reply:
You’re merging to master, have that separate oauth key setup

Reply:
added into heroku